lib/kubernetes-deploy/deferred_summary_logging.rb

```diff
@@ -34,12 +34,13 @@ def heading(text, secondary_msg = '', secondary_msg_color = :cyan)
     end
 
     # Outputs the deferred summary information saved via @logger.summary.add_action and @logger.summary.add_paragraph
-    def print_summary(success)
-      if success
-        heading("Result: ", "SUCCESS", :green)
+    def print_summary(status)
+      status_string = status.to_s.humanize.upcase
+      if status == :success
+        heading("Result: ", status_string, :green)
         level = :info
       else
-        heading("Result: ", "FAILURE", :red)
+        heading("Result: ", status_string, :red)
```

Comment:
Would it be more consistent to make this piece of heading yellow? (Just a thought; I don't have a strong opinion)

![image](https://user-images.githubusercontent.com/4789493/37979907-c9dd9e1e-31b7-11e8-87c9-d2078f27b6bc.png)